app/logic/Mail/ActiveSync/ActiveSyncEMail.ts

```diff
@@ -191,34 +191,6 @@ export class ActiveSyncEMail extends EMail {
     await super.sendInvitationResponse(response); // needs 16.x to do this automatically
     await this.deleteMessageLocally(); // Exchange deletes the message from the inbox
   }
-
-  async loadEvent() {
```

Comment:
Are you sure that they will never ever be needed anymore, for nothing?

I'd rather keep the code, disabled, by renaming the function to `loadEvent_disabled()`, and adding a comment why we don't use it, and that this will now implicitly use the implementation from `EMail.loadEvent()`, which uses iTIP.